content/redirect.js

```diff
@@ -41,6 +46,12 @@ function resolveDocumentPath(url) {
 
   const root = locale === "en-us" ? CONTENT_ROOT : CONTENT_TRANSLATED_ROOT;
 
+  if (!root) {
+    console.log(
+      `Trying to resolve a non en-us path for ${url} with out CONTENT_TRANSLATED_ROOT set.`
+    );
+    return `$TRANSLATED/${relativeFilePath}`;
```

Comment:
the _en-US_ `_redirects.txt` contains redirects to translated content. In case we don't have that checked out we still want this to work.